src/util/color/predefinedcolorpalettes.cpp

```diff
@@ -144,6 +144,21 @@ constexpr mixxx::RgbColor kVirtualDJTrackColorBlue(0x0000FF);
 constexpr mixxx::RgbColor kVirtualDJTrackColorFuchsia(0xFF00FF);
 constexpr mixxx::RgbColor kVirtualDJTrackColorWhite(0xFFFFFF);
 
+// Default Mixxx Key Color Palette
+
+constexpr mixxx::RgbColor kMixxxKeyColor1(0xFC4949);
+constexpr mixxx::RgbColor kMixxxKeyColor2(0xFE642D);
+constexpr mixxx::RgbColor kMixxxKeyColor3(0xF98C27);
+constexpr mixxx::RgbColor kMixxxKeyColor4(0xFED600);
+constexpr mixxx::RgbColor kMixxxKeyColor5(0x99FE00);
+constexpr mixxx::RgbColor kMixxxKeyColor6(0x42FE3E);
+constexpr mixxx::RgbColor kMixxxKeyColor7(0x0AD58F);
+constexpr mixxx::RgbColor kMixxxKeyColor8(0x0AE7E7);
+constexpr mixxx::RgbColor kMixxxKeyColor9(0x04C9FE);
+constexpr mixxx::RgbColor kMixxxKeyColor10(0x3D8AFD);
+constexpr mixxx::RgbColor kMixxxKeyColor11(0xAC64FE);
+constexpr mixxx::RgbColor kMixxxKeyColor12(0xFD3FEA);
+
```

Comment:
as a follow up, it would be nice if the palette was customizable via the `Color` Preferences. similar to track colors.

Reply:
Super nice to have, follow up²: set the colors in the notation table in Key preferences

Reply:
Yeah, Though I guess we'll have to decide on whether its better to put in the Color or the Key settings. The former would be more consistent IMO.

Reply:
Here's a little sketch:

![image](https://github.com/mixxxdj/mixxx/assets/57069381/0cf4a0a8-501a-425e-b726-07f2bc0a1e1e)

Reply:
I'd somehow try to put it in the same column. but I'm not a UI designer. Wdyt @ronso0?

Reply:
I think it looks more compact:

![image](https://github.com/mixxxdj/mixxx/assets/57069381/d2d9fb2f-6384-4839-8a39-a9eb7c7b1ace)

Reply:
maybe swap the order of the text and the check box so its all still right aligned? idk, IIANA Designer.